infrastructure/static/js/mm-webflow.js

```diff
@@ -686,7 +697,29 @@ function mmWebflow(js) {
       }
     });
 
-    triggerSearch();
+    function onPopstate(event) {
+      loadSearchStateFromCurrentURL();
+      triggerSearch(listView.search.createUrl(), false);
+    }
+
+    function loadSearchStateFromCurrentURL() {
+      const queryString = window.location.search.substring(1);
```

Comment:
Browsers have a good, reliable URL params method now and we don't need to chop it up ourselves. `URLSearchParams` https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams

Reply:
Changed. All I did was remove this nasty, redundant bit that manually chops up the queryString. The URLSearchParams approach is used inside `listView.loadSearchFromUrl(queryString);`